tests/tests_adapters/test_adapter_backward_compability.py

```diff
@@ -0,0 +1,48 @@
+import json
```

Comment:
Why is this module in a separate "test_adapters" folder? Also, why is it marked as new instead of renamed/ moved?

Reply:
This file was unintentionally duplicated in this folder. I deleted it.